components/base/Button.tsx

```diff
@@ -56,12 +61,20 @@ const ButtonStyle = styled.button`
     padding: 15px 25px;
     font-size: 16px;
   }
+  &.block {
+    padding: 10px 15px;
+    font-size: 14px;
+    display: block;
+    width: 100%;
+    text-align: center;
+  }
 
   &.primary {
     background-color: ${PURPLE_PRIMARY_BUTTON};
     border-color: transparent;
 
-    :hover {
+    :hover,
+    :focus {
```

Comment:
Not having a focus state reduced keyboard accessibility. Applying the hover state isn't the ideal solution, but it was a quick fix. Should probably be re-addressed later